include/zephyr/net/wifi_mgmt.h

```diff
@@ -768,6 +777,105 @@ struct wifi_ap_config_params {
 	uint32_t max_num_sta;
 };
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_DPP
```

Comment:
No need to hide these enums, structs and defines behind the Kconfig option, it is much easier to work with the code if these can be freely used.